### ISSUE TITLE
Unable to reproduce boxplot shown in website

### ISSUE THREAD

**Issue by rsquaredacademy** [jbkunst/highcharter#335]
Unable to reproduce the boxplot shown in shortcut section of the website. The `hcboxplot()` function returns an error message.

```
> data(diamonds, package = "ggplot2")
> hcboxplot(x = diamonds$x, var = diamonds$color,
+           name = "Length", color = "#2980b9")
Error in mutate_impl(.data, dots) : 
  Column `data` must be length 1 (the group size), not 5
```

I tried using a different data set but the same error message is returned.

```
> hcboxplot(x = mtcars$mpg, var = mtcars$cyl)
Error in mutate_impl(.data, dots) : 
  Column `data` must be length 1 (the group size), not 5

> hcboxplot(x = mtcars$mpg, var = as.factor(mtcars$cyl))
Error in mutate_impl(.data, dots) : 
  Column `data` must be length 1 (the group size), not 5
```

```
> sessionInfo()
R version 3.4.0 (2017-04-21)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_India.1252  LC_CTYPE=English_India.1252   
[3] LC_MONETARY=English_India.1252 LC_NUMERIC=C                  
[5] LC_TIME=English_India.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] highcharter_0.5.0 dplyr_0.7.0      

loaded via a namespace (and not attached):
 [1] igraph_1.0.1      Rcpp_0.12.11      magrittr_1.5      mnormt_1.5-5     
 [5] lattice_0.20-35   R6_2.2.1          rlang_0.1.1       TTR_0.23-1       
 [9] stringr_1.1.0     plyr_1.8.4        xts_0.9-7         tools_3.4.0      
[13] quantmod_0.4-9    parallel_3.4.0    grid_3.4.0        data.table_1.10.4
[17] broom_0.4.1       nlme_3.1-125      psych_1.6.9       htmltools_0.3.5  
[21] lazyeval_0.2.0    yaml_2.1.14       assertthat_0.2.0  digest_0.6.10    
[25] tibble_1.3.3      purrr_0.2.2.2     reshape2_1.4.2    tidyr_0.6.0      
[29] htmlwidgets_0.8   rlist_0.4.6.1     glue_1.1.0        stringi_1.1.2    
[33] compiler_3.4.0    jsonlite_1.5      lubridate_1.5.6   foreign_0.8-67   
[37] zoo_1.7-13 
```

**Comment by jbkunst** [comment]
Hello @rsquaredacademy , this was a known issue, fixed in #323 ;)!

**Comment by rsquaredacademy** [comment]
sorry @jbkunst. I should have tried with the development version.
